gst/nnstreamer/tensor_if/gsttensorif.c

```diff
@@ -242,134 +349,362 @@ static void
 gst_tensor_if_get_property (GObject * object, guint prop_id,
     GValue * value, GParamSpec * pspec)
 {
-  /* GstTensorIf *self = GST_TENSOR_IF (object); */
-  /** @todo NYI! */
+  GstTensorIf *self = GST_TENSOR_IF (object);
+  g_critical ("Getting property for prop %d.\n", prop_id);
+
+  switch (prop_id) {
+    case PROP_CV:
+      g_value_set_enum (value, self->cv);
+      break;
+    case PROP_CV_OPTION:
+      break;
+    case PROP_OP:
+      break;
+    case PROP_SV:
+      break;
+    case PROP_SV_OPTION:
+      break;
+    case PROP_THEN:
+      break;
+    case PROP_THEN_OPTION:
+      break;
+    case PROP_ELSE:
+      break;
+    case PROP_ELSE_OPTION:
+      break;
+    case PROP_SILENT:
```

Comment:
No actions for get_prop?

Reply:
Updated.
Supplied value option is not included in this stage, I will add it at #2772 .